content/docs/02.5-architecture/04.5-palette-public-ips.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "IP Addresses"
+metaTitle: "IP Addresses"
+metaDescription: "Palette's public IP Addresses."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+ 
+ # Overview
+
+ In this section, you can find the public IP addresses that support Spectro Cloud SaaS operations. These IP addresses are essential for ensuring seamless communication between your infrastructure and our platform.
```

Comment:
```suggestion
 In this section, you can find the public IP addresses that support Spectro Cloud SaaS operations. These IP addresses are essential to ensure seamless communication between your infrastructure and our platform.
```

Reply:
@karl-cardenas-coding The sentence is fine as is, and this very minor suggestion is totally up to you @karl-cardenas-coding.